pybv/io.py

```diff
@@ -34,8 +33,8 @@
 }
 
 
-def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
-                      events=None, resolution=0.1, unit='µV',
+def write_brainvision(*, data, sfreq, ch_names, ref_ch_names=None, fname_base,
+                      folder_out, events=None, resolution=0.1, unit='µV',
```

Comment:
> honest question: is it good practice to have default args and non-default args intermixed? It seems to work here due to the leading `*` arg, but it looks a bit odd and without the leading `*` it would raise a SyntaxError.

I don't know if it's considered good practice, but I think it's one of the big advantages of using kw-only params to be able to do this :) I think it makes **a lot** of sense to have `ref_ch_names` directly after `ch_names`, even though it's not a required parameter.

Reply:
agreed, let's do it this way then :+1:

Reply:
I've never seen this either, it will certainly cause some confusion. Is there any alternative other than putting all default args at the end? And yes, because of the `*` there is no "end" because these are all kw-only args.

Reply:
> Is there any alternative other than putting all default args at the end?

None that wouldn't be even more confusing: assign a default value like `None` to parameters that need to be set. 🤷‍♂️